components/OpenApplicationsView.tsx

```diff
@@ -22,7 +22,8 @@ export default function OpenApplicationsView({
   typeformApplications,
   userData,
 }: OpenApplicationsViewProps) {
-  return typeformApplications[0] && applications ? (
+  console.log(applications);
+  return typeformApplications.length !== 0 && applications.length !== 0 ? (
```

Comment:
Should be `||` instead of `&&` here. Also remove `console.log`. 

- [ ] Change condition type
- [ ] Remove `console.log`